### PR TITLE
feat(OkuSeparator): create new component

### PR DESCRIPTION
**Pull request by akintoluvic**
Adds new component feature OkuSeparator

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -48,4 +48,5 @@
         },
     ],
     "references.preferredLocation": "peek",
+    "typescript.tsdk": "node_modules/typescript/lib",
```
Comment (productdevbook) [inline .vscode/settings.json:51]: why add ?

Comment (akintoluvic) [reply .vscode/settings.json:51]: That was added automatically after linting

Comment (productdevbook) [reply .vscode/settings.json:51]: okay, please delete

Comment (akintoluvic) [reply .vscode/settings.json]: I have removed it

**Review by productdevbook — APPROVED** [review]

**Pull request merged**